src/utils/validationUtils.ts

```diff
@@ -0,0 +1,11 @@
+const validateURL = (_url: string): boolean => {
+  let url
+  try {
+    url = new URL(_url)
+  } catch (_) {
+    return false
+  }
+  return url.protocol === 'http:' || url.protocol === 'https:'
```

Comment:
I think it would be better to use a constant for the valid protocols. Also, we're trying to keep the use of mutable variables close to none. Another point is to try not to use arg names with underscore prefix. In fact I'm surprised eslint didn't complain about that. I suggest following structure:

```suggestion
const EVENT_API_AVAILABLE_PROTOCOLS = [ 'http:', 'https:'] // we could potentially make this configurable, for sure we should move it somewhere more in scope of the notifier service

const validateURLProtocol = (url: string, validProtocols: Array<string>): boolean => {
  try {
    const { protocol } = new URL(url)
    return validProtocols.includes(protocol);
  } catch { // note that catch in js doesn't need arguments
    return false
  }
}
```